AstraEngine.Serialization/AstraEngine.Serialization.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>

```

Comment:
@maxwaldman8 
When you intend other people to use your code as a library, you should include this line in the `<PropertyGroup>` section: `<GenerateDocumentationFile>true</GenerateDocumentationFile>`.

You can see this here:

https://github.com/maxwaldman8/AstraEngine/blob/3-Feature-Request-YAML-Parser/AstraEngine.InputSystem/AstraEngine.InputSystem.csproj

```xml
﻿<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net8.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
  </PropertyGroup>

</Project>
```